data/language/english_uk.txt

```diff
@@ -4039,8 +4038,20 @@ STR_5732    :Anisotropic
 STR_5733    :Use NN scaling at integer scales
 # tooltip for tab in options window
 STR_5734    :{SMALLFONT}{BLACK}Rendering
-STR_5735    :Network Status
-STR_5736    :Player
```

Comment:
I'll keep that in mind, it's a good idea once I know what strings I need.  I somehow broke my dev installation of ORCT so that's why this is taking a while.